src/analysis/retail/paladin/holy/modules/checklist/Component.tsx

```diff
@@ -37,10 +37,10 @@ const HolyPaladinChecklist = ({ combatant, castEfficiency, thresholds }: Checkli
             is your most efficient healing spell available. Try to cast them as much as possible
             without overhealing.
             <TooltipElement
-              content={t({
+              content={{
                 id: 'paladin.holy.modules.checklist.usePrimarySpells.descriptionTooltipElement',
                 message: `When you're not bringing too many healers.`,
-              })}
+              }}
```

Comment:
is this correct? i don't think we can pass bare objects in and have them translated

Reply:
My bad, resolved.